src/linear_algebra/dfinsupp.lean

```diff
@@ -231,6 +232,29 @@ begin
     exact dfinsupp_sum_add_hom_mem _ v _ (λ i _, (le_supr p i : p i ≤ _) (v i).prop) }
 end
 
+/-- The bounded supremum of a family of commutative additive submonoids is equal to the range of
+`dfinsupp.sum_add_hom` composed with `dfinsupp.filter_add_monoid_hom`; that is, every element in the
+bounded `supr` can be produced from taking a finite number of non-zero elements from the `S i` that
+satisfy `p i`, coercing them to `γ`, and summing them. -/
+lemma bsupr_eq_mrange_dfinsupp_lsum (p : ι → Prop)
```

Comment:
`mrange` is the range as an (`add_`)`monoid` (in contrast to `range` which is a group, so it makes no sense to add the `m` here:
```suggestion
lemma bsupr_eq_range_dfinsupp_lsum (p : ι → Prop)
```

Reply:
Whoops, good catch